gcloud-java-datastore/pom.xml

```diff
@@ -34,12 +34,12 @@
     <dependency>
       <groupId>com.google.cloud.datastore</groupId>
       <artifactId>datastore-v1beta3-protos</artifactId>
-      <version>0.0.1-SNAPSHOT</version>
+      <version>1.0.0-beta-SNAPSHOT</version>
```

Comment:
Before, the artifact was only available from the sonatype snapshot repository and not the maven central repository.  I've modified it and rebased.